core/domain/exp_services.py

```diff
@@ -250,13 +311,26 @@ def get_multiple_explorations_by_id(exp_ids, strict=True):
 
 
 def get_new_exploration_id():
-    """Returns a new exploration id."""
+    """Returns a new exploration id.
+
+    Returns:
+        A new exploration id.
+    """
     return exp_models.ExplorationModel.get_new_id('')
 
 
 def is_exp_summary_editable(exp_summary, user_id=None):
     """Checks if a given user may edit an exploration by checking
     the given domain object.
+
+    Args:
+        exp_summary: A domain object of exploration summary.
+        user_id: id of the user.
+
+    Returns:
+        True or False by checking if given user has edited an
```

Comment:
No -- this is actually to check rights, not what they've done before.

Returns:
    bool. Whether the given user has edit permissions for the given exploration.